packages/notifier/src/index.js

```diff
@@ -26,3 +28,6 @@ export {
 } from './asyncIterableAdaptor.js';
 export * from './storesub.js';
 export * from './stored-notifier.js';
+
+// eslint-disable-next-line import/export -- doesn't know types
+export * from './types.js';
```

Comment:
so this is a solution to the question of re-exporting?

Reply:
this is the right way to export at the top level. unfortunately it doesn't work for `@agoric/ertp` because it has `AssetKind` as both a type and kind